packages/web/pages/index.js

```diff
@@ -23,25 +28,68 @@ const Home = ({ emailConfirmation, changeEmail, technologies }) => {
 	return (
 		<>
 			<Hero />
-			{!!technologies?.featured?.length && (
-				<TechnologiesSection
-					header={t('common:featuredSolutions')}
-					technologies={technologies.featured}
-					bgColor={colors.whiteSmoke}
-				/>
+			<ButtonsWrapper>
+				<ButtonsContainer>
+					<RectangularButton
+						as="a"
+						href="/announcements-bank"
+						target="_blank"
+						variant="backgroundImage"
+						backgroundUrl="/buttons/papers-background.png"
+						colorVariant="green"
+					>
+						Banco de editais
+					</RectangularButton>
+					<RectangularButton
+						as="a"
+						href="/ideas-bank"
+						target="_blank"
+						variant="backgroundImage"
+						backgroundUrl="/buttons/paper-light.png"
+						colorVariant="orange"
+					>
+						Banco de ideias
+					</RectangularButton>
+					<RectangularButton
+						as="a"
+						href="/researchers-bank"
+						target="_blank"
+						variant="backgroundImage"
+						backgroundUrl="/buttons/notebook-writing.png"
+						colorVariant="blue"
+					>
+						Banco de pesquisadores
+					</RectangularButton>
+				</ButtonsContainer>
+			</ButtonsWrapper>
+
+			{technologies?.featured?.length && (
+				<TechnologiesSection>
+					<SolutionsSection
+						header={t('common:featuredTechnologies')}
+						data={technologies.featured}
+						bgColor={colors.lightGray4}
+						type="technology"
+						padding="0rem 5%"
+					/>
+				</TechnologiesSection>
 			)}
-			{!!technologies?.recent?.length && (
-				<TechnologiesSection
-					header={t('common:recentSolutions')}
-					technologies={technologies.recent}
-					bgColor={colors.whiteSmoke}
+
+			{services.length && (
+				<SolutionsSection
+					header={t('common:featuredServices')}
+					data={services}
+					bgColor={colors.lightGray4}
+					type="service"
 				/>
 			)}
 		</>
 	);
 };
 
 Home.getInitialProps = async ({ req }) => {
+	resetIdCounter();
```

Comment:
It's already in `_app.js`, I guess

Reply:
I put it in `_document`, just forgot to remove from this page 👍🏻